src/renault_api/const.py

```diff
@@ -0,0 +1,182 @@
+"""Constants for Renault API."""
+CONF_GIGYA_APIKEY = "gigya-api-key"
+CONF_GIGYA_URL = "gigya-api-url"
+CONF_KAMEREON_APIKEY = "kamereon-api-key"
+CONF_KAMEREON_URL = "kamereon-api-url"
+
+AVAILABLE_LOCALES = {
+    "bg_BG": {
+        CONF_GIGYA_URL: "https://accounts.eu1.gigya.com",
```

Comment:
I think we can have a const for `GIGYA_URL_EU` (28 times here) and `GIGYA_URL_US`